plugins/outputs/opentsdb/opentsdb_http.go

```diff
@@ -123,7 +124,7 @@ func (o *openTSDBHttp) flush() error {
 		Scheme: o.Scheme,
 		User:   o.User,
 		Host:   fmt.Sprintf("%s:%d", o.Host, o.Port),
-		Path:   "/api/put",
+		Path:   o.Path,
```

Comment:
Can you default `o.Path` to `"/api/put"` in order to not break any current implementations.